backend/hitas/oracle_migration/utils.py

```diff
@@ -80,6 +81,18 @@ def housing_company_state_from(code: str) -> HousingCompanyState:
             return HousingCompanyState.HALF_HITAS
 
 
+def housing_company_regulation_status_from(code: str) -> RegulationStatus:
+    # These are hardcoded as the code number (C_KOODISTOID) and
+    # the name (C_NAME) are the only ones that can be used to
+    # identify these types
+    if code == "003":
+        return RegulationStatus.RELEASED_BY_HITAS
+    if code == "004":
+        return RegulationStatus.RELEASED_BY_PLOT_DEPARTMENT
+
+    return RegulationStatus.REGULATED
```

Comment:
Should `005` (`Half-Hitas`) be regulated or not? I'm not 100% sure which one it should be 🤔

Reply:
If I remember correctly, half-Hitas housing companies are only regulated for the initial sale, and not afterward.